R/session-cookie.R

```diff
@@ -40,6 +40,11 @@
 #'   Defaults to \code{TRUE}.
 #' @param secure Boolean that adds the \code{Secure} cookie flag.  This should be set
 #'   when the route is eventually delivered over \href{https://en.wikipedia.org/wiki/HTTPS}{HTTPS}.
+#' @param sameSite A character specifying the SameSite policy to attach to the cookie.
+#'   If specified, one of the following values should be given: "Strict", "Lax", or "None".
+#'   If "None" is specified, then the \code{secure} flag MUST also be set for the modern browsers to
+#'   accept the cookie.
+#'   If not specified or a non-character is given, no SameSite policy is attached to the cookie.
```

Comment:
Can we implement this logic within `sessionCookie`?

Ex:
```r
if (is.character(sameSite)) {
  sameSite <- match.arg(sameSite, c("Strict", "Lax", "None"))
}
if (identical(sameSite, "None")) {
  if (!secure) {
    stop("informative message here")
  }
}
```

Reply:
Just added that logic into `sessionCookie`, also updated the documentation for `sessionCookie` and `pr_cookie` to note that an error will be raised if that particular set of parameters is given.